src/views/datasets/booster-gap/dashboard.vue

```diff
@@ -117,95 +117,7 @@
   <DashboardCard width="full">
     <template #title>What can I do to close the gap?</template>
     <template #content>
```

Comment:
I was having trouble doing that because the template is inside the content template and I was having issues moving it! Let me try and see what happens

Reply:
Ah I see! Thanks for taking a look! We can ignore my comment then!